core/src/instance.rs

```diff
@@ -74,14 +74,21 @@ impl Instance {
         )
     }
 
-    /// Start the Event Loop on a seperate thread
-    pub fn start_action_loop(&mut self, context: Arc<Context>) {
+    /// Returns recievers for actions and observers that get added to this instance
+    fn start_listening(&mut self) -> (Receiver<ActionWrapper>, Receiver<Observer>) {
```

Comment:
"start listening" doesn't mean anything to me

"add/init/build channels" is what we're doing here right?

Reply:
I'll change this.